examples/call-grpc.yaml

```diff
@@ -8,7 +8,7 @@ do:
       call: grpc
       with:
         proto: 
-          endpoint: file://app/greet.proto
+          endpoint: https://greeter.com/greet.proto
```

Comment:
I mean there are many other valid protocols, such as `ftp://`. There is no blocker to adding them to the pattern.

But similar to the discussions around containers I don't think there is a defined way for working with the file system yet. Because in a distributed engine where would that file come from?

I would prefer restricting it to http(s) for now and coming up with a defined way to interact with the file system

Reply:
@matthias-pichler good point, as usual!

Reply:
As discussed during today's meeting, could you update the regex to not restrict to specific schemes? Even if your rationale makes perfect sense (at least to me), our user base might have edge use cases where other schemes are required, as pointed out by @ricardozanini and others.

Reply:
We could use something like the following regex:
```regex
^[a-zA-Z][a-zA-Z0-9+.-]*://.*
```

Reply:
I will add a general spec part according to the uri spec instead of just http(s) 👍